site/src/comparison.rs

```diff
@@ -1222,19 +1220,30 @@ impl std::hash::Hash for TestResultComparison {
 // The direction of a performance change
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
 pub enum Direction {
+    None,
     Improvement,
     Regression,
     Mixed,
 }
 
-impl std::fmt::Display for Direction {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        let description = match self {
-            Direction::Improvement => "improvement",
-            Direction::Regression => "regression",
-            Direction::Mixed => "mixed",
-        };
-        write!(f, "{}", description)
+// The direction of a performance change. Forms a lattice:
+//
+//          Mixed
+//          /    \
+// Improvement  Regression
+//          \    /
+//           None
+//
+impl Direction {
+    // Also known as the "least upper bound".
+    pub fn join(a: Self, b: Self) -> Self {
```

Comment:
Nit: `combine` feels like a better name than `join` but I don't feel strongly about it.

Reply:
"join" is the standard term from lattice theory: https://en.wikipedia.org/wiki/Join_and_meet